### PR TITLE
check this out!!

### PR DESCRIPTION
**Pull request by milleymilli**

### CODE REVIEW

File: js/my-github.js [hunk 1]
```diff
@@ -1 +1,70 @@
...
+var ownerName = document.getElementById("reposOwner");
+fetch("https://api.github.com/users/milleymilli/repos")
+  .then(response => {
+    return response.json();
+  })
+  .then(data => {
+    // console.log(data.length);
+    xClick.innerHTML = data.length;
+    let reposi = "";
+    data.forEach(element => {
```
Comment (jslopezgithub) [inline js/my-github.js:11]: This is correct. This is an alternative approach which uses an external function.
```suggestion
            json.forEach(element => {
                 const item = createItem(element.full_name, element.html_url);
                 repoList.appendChild(item);
```

File: js/my-github.js [hunk 2]
```diff
@@ -1 +1,70 @@
...
+    //console.log(reposi);
+    document.querySelector("#repos-list li").innerHTML = reposi;
+  })
+  .catch(error => {
+    console.log(error);
+  });
+
+//getting github by using searching button
+
+var labeluser = document.createElement("label");
```
Comment (jslopezgithub) [inline js/my-github.js:25]: This code should be in my-github.html instead.

File: js/my-github.js [hunk 3]
```diff
@@ -1 +1,70 @@
...
+searchButton.style.marginLeft = "5px";
+
+labeluser.appendChild(searchButton);
+
+searchButton.addEventListener("click", () => {
+  var usernName = input.value;
+  fetchingUser(usernName);
+});
+
+function fetchingUser(name) {
```
Comment (jslopezgithub) [inline js/my-github.js:50]: 💯 good use of a helper function

**Review by jslopezgithub — APPROVED** [review]
Looks good

File: js/my-github.js (hunk 1 above)
Comment (milleymilli) [reply js/my-github.js:11]: yes i can see ,,,but won't be a link after u append it. It should be a link so we have to put element.html_url in href attribute i guess

Comment (jslopezgithub) [reply js/my-github.js:11]: Yes, it is a link after appending it. This is the code for createItem.
```
function createItem(name, url) {
    const item = document.createElement("li");
    const link = document.createElement("a");
    item.appendChild(link);
    link.textContent = name;
    link.setAttribute("href", url);
    return item;
}
```

**Pull request closed without merging**